examples/complete/variables.tfvars

```diff
@@ -0,0 +1,9 @@
+enabled = true
```

Comment:
as there is no need for variables so there is no need also .tfvars in examples,
keep example as minimal as possible so that one will be easily navigate/see what/how works,
you can have this values in locals{} or even use it directly in module